pythonforandroid/recipes/argon2-cffi/__init__.py

```diff
@@ -0,0 +1,11 @@
+from pythonforandroid.recipe import CompiledComponentsPythonRecipe
+
+
+class Argon2Recipe(CompiledComponentsPythonRecipe):
+    version = '20.1.0'
+    url = 'git+https://github.com/hynek/argon2-cffi@{version}'
```

Comment:
I think you need to remove the `@{version}` here. Sorry.

Reply:
np.